### ISSUE TITLE
CAD-707 ChainDB.metrics:Critical messages when syncing  

### ISSUE THREAD

**Issue by oneEdoubleD** [input-output-hk/cardano-node#618]
When syncing `tags/1.7.0`, there are numerous `ChainDB.metrics:Critical` messages. I'd assume these metrics would be `Info` rather than `Critical` as they are written for every `slot`. 

Example: 

```
[edward:cardano.node.ChainDB:Info:36] [2020-03-02 10:20:16.16 UTC] {"event":{"kind":"TraceAddBlockEvent.AddBlockValidation.ValidCandidate","block":"b5f0ecb4447947e0@1259"},"kind":"TraceEvent","tip":"29e09b2@1258"}
[edward:cardano.node.ChainDB.text:Info:36] [2020-03-02 10:20:16.16 UTC] [29e09b2@1258] Valid candidate (Point 1259, b5f0ecb4447947e0)
[edward:cardano.node.ChainDB.metrics:Critical:36] [2020-03-02 10:20:16.16 UTC] density = 1.0
[edward:cardano.node.ChainDB.metrics:Critical:36] [2020-03-02 10:20:16.16 UTC] slotNum = 1259
[edward:cardano.node.ChainDB.metrics:Critical:36] [2020-03-02 10:20:16.16 UTC] blockNum = 1260
[edward:cardano.node.ChainDB.metrics:Critical:36] [2020-03-02 10:20:16.16 UTC] slotInEpoch = 1259
[edward:cardano.node.ChainDB.metrics:Critical:36] [2020-03-02 10:20:16.16 UTC] epoch = 0
[edward:cardano.node.ChainDB:Notice:36] [2020-03-02 10:20:16.16 UTC] {"event":{"kind":"TraceAddBlockEvent.AddedToCurrentChain","newtip":"b5f0ecb4447947e0@1259"},"kind":"TraceEvent","tip":"29e09b2@1258"}
[edward:cardano.node.ChainDB.text:Notice:36] [2020-03-02 10:20:16.16 UTC] [29e09b2@1258] Chain extended, new tip: (Point 1259, b5f0ecb4447947e0)
[edward:cardano.node.ChainDB:Debug:36] [2020-03-02 10:20:16.16 UTC] {"event":{"kind":"TraceAddBlockEvent.AddedBlockToVolDB","blockNo":"1261","block":{"hash":"fe8d81eae73fc213","kind":"Point","slot":1260}},"kind":"TraceEvent","tip":"b5f0ecb@1259"}
```

**Comment by disassembler** [comment]
these should not be logged at all but rather just utilized by prometheus/ekg.

**Comment by CodiePP** [comment]
https://jira.iohk.io/browse/CAD-707

**Comment by frankaz** [comment]
Hello,

Was this resolved as of 1.24.2? Starting up a new node and running a test to see if the node was set up correctly. But also getting "Critical" messages. I haven't changed anything in testnet-config.json. Let it run for 1 hour to see if it would resolve, but still received numerous "critical"s after 60-70 min. Also tried changing "TraceBlockFetchDecisions" to true (as recommended in the tutorial video from 07/2020), but that didn't change the errors/warnings.

CPU was running at 25% after 60 min. This leads me to believe it might be having issues establishing a connection. Could you please let me know if this is expected and safe to move on?

Input: 
cardano-node run \
 --topology testnet-topology.json \
 --database-path db \
 --socket-path db/node.socket \
 --host-addr aws.ip.address.x \
 --port 3001 \
 --config testnet-config.json

Output:
[usr-172:cardano.node.BlockFetchDecision.peers:Info:62] [2021-03-19 02:31:15.88 UTC] connectedPeers = 2
[usr-172:cardano.node.ChainDB.metrics:Critical:52] [2021-03-19 02:31:15.92 UTC] density = 1.0
[usr-172:cardano.node.ChainDB.metrics:Critical:52] [2021-03-19 02:31:15.92 UTC] slotNum = 1210169
[usr-172:cardano.node.ChainDB.metrics:Critical:52] [2021-03-19 02:31:15.92 UTC] blockNum = 1209020
[usr-172:cardano.node.ChainDB.metrics:Critical:52] [2021-03-19 02:31:15.92 UTC] slotInEpoch = 569
[usr-172:cardano.node.ChainDB.metrics:Critical:52] [2021-03-19 02:31:15.92 UTC] epoch = 56
[usr-172:cardano.node.ChainDB.metrics:Critical:52] [2021-03-19 02:31:15.92 UTC] density = 1.0
[usr-172:cardano.node.ChainDB.metrics:Critical:52] [2021-03-19 02:31:15.92 UTC] slotNum = 1210170
[usr-172:cardano.node.ChainDB.metrics:Critical:52] [2021-03-19 02:31:15.92 UTC] blockNum = 1209021
[usr-172:cardano.node.ChainDB.metrics:Critical:52] [2021-03-19 02:31:15.92 UTC] slotInEpoch = 570
[usr-172:cardano.node.ChainDB.metrics:Critical:52] [2021-03-19 02:31:15.92 UTC] epoch = 56
[usr-172:cardano.node.ChainDB.metrics:Critical:52] [2021-03-19 02:31:15.92 UTC] density = 1.0
[usr-172:cardano.node.ChainDB.metrics:Critical:52] [2021-03-19 02:31:15.92 UTC] slotNum = 1210171
[usr-172:cardano.node.ChainDB.metrics:Critical:52] [2021-03-19 02:31:15.92 UTC] blockNum = 1209022
[usr-172:cardano.node.ChainDB.metrics:Critical:52] [2021-03-19 02:31:15.92 UTC] slotInEpoch = 571
[usr-172:cardano.node.ChainDB.metrics:Critical:52] [2021-03-19 02:31:15.92 UTC] epoch = 56
[usr-172:cardano.node.ChainDB.metrics:Critical:52] [2021-03-19 02:31:15.92 UTC] density = 1.0
[usr-172:cardano.node.ChainDB.metrics:Critical:52] [2021-03-19 02:31:15.92 UTC] slotNum = 1210172
[usr-172:cardano.node.ChainDB.metrics:Critical:52] [2021-03-19 02:31:15.92 UTC] blockNum = 1209023
[usr-172:cardano.node.ChainDB.metrics:Critical:52] [2021-03-19 02:31:15.92 UTC] slotInEpoch = 572
[usr-172:cardano.node.ChainDB.metrics:Critical:52] [2021-03-19 02:31:15.92 UTC] epoch = 56
[usr-172:cardano.node.BlockFetchDecision.peers:Info:62] [2021-03-19 02:31:15.90 UTC] connectedPeers = 2
[usr-172:cardano.node.BlockFetchDecision.peers:Info:62] [2021-03-19 02:31:15.92 UTC] connectedPeers = 2
[usr-172:cardano.node.ChainDB.metrics:Critical:52] [2021-03-19 02:31:15.98 UTC] density = 1.0
[usr-172:cardano.node.ChainDB.metrics:Critical:52] [2021-03-19 02:31:15.98 UTC] slotNum = 1210173
[usr-172:cardano.node.ChainDB.metrics:Critical:52] [2021-03-19 02:31:15.98 UTC] blockNum = 1209024
[usr-172:cardano.node.ChainDB.metrics:Critical:52] [2021-03-19 02:31:15.98 UTC] slotInEpoch = 573
[usr-172:cardano.node.ChainDB.metrics:Critical:52] [2021-03-19 02:31:15.98 UTC] epoch = 56
[usr-172:cardano.node.BlockFetchDecision.peers:Info:62] [2021-03-19 02:31:15.98 UTC] connectedPeers = 2
[usr-172:cardano.node.DnsSubscription:Error:78] [2021-03-19 02:31:16.00 UTC] [String "Application Exception: 54.151.49.138:3001 SubscriberError {seType =
 SubscriberWorkerCancelled, seMessage = \"SubscriptionWorker exiting\", seStack = []}",String "SubscriptionTrace",String "\"relays-new.cardano-testnet.ioh
kdev.io\""]
[usr-172:cardano.node.ChainDB.metrics:Critical:52] [2021-03-19 02:31:16.00 UTC] density = 1.0
[usr-172:cardano.node.ChainDB.metrics:Critical:52] [2021-03-19 02:31:16.00 UTC] slotNum = 1210174
[usr-172:cardano.node.ChainDB.metrics:Critical:52] [2021-03-19 02:31:16.00 UTC] blockNum = 1209025
[usr-172:cardano.node.ChainDB.metrics:Critical:52] [2021-03-19 02:31:16.00 UTC] slotInEpoch = 574
[usr-172:cardano.node.ChainDB.metrics:Critical:52] [2021-03-19 02:31:16.00 UTC] epoch = 56
[usr-172:cardano.node.BlockFetchDecision.peers:Info:62] [2021-03-19 02:31:16.01 UTC] connectedPeers = 0
[usr-172:cardano.node.BlockFetchDecision:Info:62] [2021-03-19 02:31:16.01 UTC] before next, messages elided = 122841
[usr-172:cardano.node.BlockFetchDecision:Info:62] [2021-03-19 02:31:16.01 UTC] [String "PeersFetch",Array [Object (fromList [("peer",String "ConnectionId
 {localAddress = 172.x.x.x:45093, remoteAddress = 54.151.49.138:3001}"),("kind",String "FetchDecision declined"),("declined",String "FetchDeclinePeerBu
sy 387960 196608 393216")]),Object (fromList [("peer",String "ConnectionId {localAddress = 172.x.x.x:40719, remoteAddress = 54.241.77.32:3001}"),("kind
",String "FetchDecision declined"),("declined",String "FetchDeclineConcurrencyLimit FetchModeBulkSync 1")])]]
[usr-172:cardano.node.DnsSubscription:Error:79] [2021-03-19 02:31:16.01 UTC] [String "Application Exception: 54.241.77.32:3001 SubscriberError {seType = 
SubscriberWorkerCancelled, seMessage = \"SubscriptionWorker exiting\", seStack = []}",String "SubscriptionTrace",String "\"relays-new.cardano-testnet.iohk
dev.io\""]
[usr-172:cardano.node.ChainDB:Info:5] [2021-03-19 02:31:16.02 UTC] Closed db with immutable tip at c1ef6f84fcd69a208864ef1e9912532835a9e3200b1f142af29a88
593f75e006 at slot 1208014 and tip 2d7c3a3dec6b86e6cea6b89090bbcf2bd39f8d3addd687188d9855e2d59cf6ac at slot 1210174
